x/bank/handler.go

```diff
@@ -54,18 +54,20 @@ func handleMsgLockedSend(ctx sdk.Context, k Keeper, ak types.AccountKeeper, msg
 		}
 	}
 
-	// subtract from sender account (as normally done)
-	_, err := k.SubtractCoins(ctx, msg.From, msg.Amount)
-	if err != nil {
-		return nil, err
-	}
-
 	// add to receiver account as normally done
 	// but make the added amount vesting (OV := Vesting + Vested)
 	toAcc.OriginalVesting = toAcc.OriginalVesting.Add(msg.Amount...)
+	newCoins := toAcc.Coins.Add(msg.Amount...)
+	if newCoins.IsAnyNegative() {
```

Comment:
When can this happen?

Reply:
Don't really know but Cosmos had them in AddCoins, so I just added it in.